cards/README.md

```diff
@@ -1,53 +1,54 @@
 # Cards
 The Card object holds information about the card details submitted by the user using the web or mobile SDKs. One user can link multiple cards, debit or credit under the same account.
 
-The details required to link a card to a program are the 16-digit card number, expiry date, country code and the `programId` of the Program you want to link this card to. The card number is tokenised and transmitted directly from our secure pre-built iFrame to the API. This way, your servers are never exposed to sensitive information, removing all PCI compliance requirements from your side.
+In order to link a card to a program, we recommend that you use our secure and PCI compliant SDKs. The user will enter their  16-digit card number, and expiry date.  The SDK will be prepopulated with the country code and the `programId` of the Program you want to link this card to. On submission, he card number is tokenised and transmitted directly from our secure pre-built iFrame to the API. This way, your servers are never exposed to sensitive information, removing all PCI compliance requirements from your side.
 
-We never store the 16-digit PAN (Primary Account Number). To identify the user in a transaction object you should the use the `cardId` property. After this point only the `cardId` is exchanged between your servers, the card schemes and Fidel API.
+We never store the 16-digit card number, but rather use the `cardId` to the user. After a user signs up, only the  `cardId` is exchanged between your servers, the card schemes and Fidel API.
```

Comment:
`use the cardId to the user` -> I feel there's a word missing here, maybe to identify the user? Although that's not even true, we can't identify a user, we can only identify a card. from `someCardId` I don't know if it's Barbara's card or Alice's, only that it's _that_ card.

Reply:
@dougsillars this is still pending

Reply:
fixed in another comment :)